pallets/subtensor/src/lib.rs

```diff
@@ -1325,7 +1335,9 @@ pub mod pallet {
                 // Storage version v4 -> v5
                 .saturating_add(migration::migrate_delete_subnet_3::<T>())
                 // Doesn't check storage version. TODO: Remove after upgrade
-                .saturating_add(migration::migration5_total_issuance::<T>(false));
+                .saturating_add(migration::migration5_total_issuance::<T>(false))
+                // Populate Owned map for coldkey swap. Doesn't update storage vesion.
+                .saturating_add(migration::migrate_populate_owned::<T>());
```

Comment:
Do we know the number of storage read/writes here?

Reply:
Migration Populate OwnedHotkeys map finished. Keys touched: 77299, Longest hotkey vector: 1000, Storage reads: 154598, Storage writes: 77299